src/components/BarChartCard/BarChartCard.jsx

```diff
@@ -78,14 +78,15 @@ const BarChartCard = ({
   const size = increaseSmallCardSize(sizeProp, 'BarChartCard');
 
   // If editable, show sample presentation data
-  const values = isEditable
```

Comment:
PR open for these changes: https://github.com/carbon-design-system/carbon-addons-iot-react/pull/1686